### ISSUE TITLE
Allow functions to be defined in another clj file and required or imported

### ISSUE THREAD

**Issue by retrogradeorbit** [epiccastle/spire#25]

**Comment by retrogradeorbit** [comment]
released in 0.1.0-alpha.6

**Comment by michaelglass** [comment]
this works for us but is failing when we have nested dependencies.

e.g. top-level requires work, but requires from required files do not find the required file.


`src/script.clj` can require src/utils/util.clj`
but `src/utils/util.clj` fails to find `src/utils/log.clj` 
```
Exception in thread "main" clojure.lang.ExceptionInfo: Could not find namespace: util.log. {:type :sci/error, :line 2, :column 3, :message "Could not find namespace: util.log.", :sci.impl/callstack #object[clojure.lang.Volatile 0x4f298af2 {:status :ready, :val ({:line 2, :column 3, :file "src/utils/util.clj", :ns #object[sci.impl.vars.SciNamespace 0x19ffedf7 "utils.util"]})}], :file "src/utils/util.clj", :locals {}}
	at sci.impl.utils$rethrow_with_location_of_node.invokeStatic(utils.cljc:85)
	at sci.impl.evaluator$eval.invokeStatic(evaluator.cljc:338)
```
